projects/arc/src/environments/environment.ts

```diff
@@ -0,0 +1,10 @@
+export const environment = {
+  production: false,
+  clientId: '',
+  publicKey: '',
+  homePath: '/main/home',
+  baseApiUrl: '',
+  authServiceUrl: '',
+  userServiceUrl: '',
+  logLevel: 5,
```

Comment:
are we using env resolver for this ?

Reply:
currently we are not using it we are using injection_token for the env